### DISCUSSION TITLE
Add opening hours to Places schema?

### DISCUSSION THREAD

**Discussion by eisams** [OvertureMaps/data#298] [Discussion]
I had a look at the Places dataset and I couldn't find that there's a property for opening hours?

This would be very useful for data consumers, and at least the Meta sourced dataset should contain it

Would there be a plan to introduce it? I noticed there's a [openingHours property at the top-level](https://github.com/OvertureMaps/schema/blob/581aefaa132ed6a896edbfb4ec2fe1d78e58d2a2/schema/defs.yaml#L149), but it's only used in the Transportation theme

**Comment by merylstreet** [comment]
It would be very useful to have this info included in Overture Places, many small shops owners regularly update their opening hours on their Facebook page

**Comment by eisams** [comment]
I guess the answer is here, there's no plans to include it. You'd need to connect with someone offering it as proprietary data?

https://lwn.net/Articles/995992/
> Returning to GERS, Prioleau said that mapping is being driven by an explosion of data types. But only some types are "exploding". The road network, while constantly changing, is relatively easy to keep up with; the same is true of addresses and such. This kind of data is manageable as a shared base layer that all can contribute to and use. The attached layers, though, which might include traffic data, restaurant reviews, **opening hours, or any other sort of add-on data, are far too volatile to be managed in an open layer. This data will, he said, continue to be a proprietary product indefinitely.**

**Comment by ChenemiAbraham** [comment]
Back to this thread but later... the Open Hours data can be figured out via Facebook definitely.
